mobile/src/components/interview/components/other-applications/other-applications.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+
+import { CourseGetRequestParamsDto } from '~/common/types/types';
+import { Stack, Text, View } from '~/components/common/common';
+import { useAppDispatch, useAppSelector, useEffect } from '~/hooks/hooks';
+import { interviewActions } from '~/store/actions';
+
+import { ApplicationCard } from './components/components';
+import { styles } from './styles';
+
+const OtherApplications: FC = () => {
+  const dispatch = useAppDispatch();
+  const { otherInterviews, totalOtherInterviewsNumber } = useAppSelector(
+    (state) => state.interview,
+  );
+
+  useEffect(() => {
+    dispatch(
+      interviewActions.getOtherByInterviewId({
+        // for now id 2 after we get id from state (Dmitrij Revenets PR GRHB-356)
+        interviewId: 2,
+        // Maybe we don't need pagination on mobile but we have
+        // in shared folder type InterviewsGetOtherRequestDto with pagination.
+        // Create new type or leave it like that ?
+        page: 1,
+        count: totalOtherInterviewsNumber,
```

Comment:
Stay with pagination, do we have a default constant for page and count?

Reply:
No for now we don't have any constants

Reply:
Maybe you should use PaginationDefaultValue from enums for it?

Reply:
If add PaginationDefaultValue, then we probably need to display cards through flatlist and load cards when scrolling. I'm not sure we need it here

Reply:
Sorry, I didn't notice that this is a table on the frontend, rewrite it to a table, please